epub33/core/index.html

```diff
@@ -11497,7 +11497,13 @@ <h3>The <code>application/oebps-package+xml</code> media type</h3>
 
 					<dt>Encoding considerations:</dt>
 					<dd>
-						<p>Package documents are UTF-8 or UTF-16 encoded XML.</p>
+						<p>8bit if UTF-8; binary if UTF-16.</p>
+						<p>
+							Package documents are in XML, represented either in UTF-8 or UTF-16. When the 
+							package document is written in UTF-8, the file is 8bit compatible.  
+							When it is written in UTF-16, the binary content-transfer-encoding
```

Comment:
RFC7303 says it must be base64 in s. 9.1, so while that is a form of binary, is "binary" technically correct here?

I don't know what to suggest for this given the four values they seem to want. All the other +xml registrations I've looked at either defer to rfc 3023 or 7303, depending on their age, without getting into these kinds of details.

Reply:
Isn't it nice to see this consistency across rfc-s? 

7303 says

> ...but binary data, for example, data with a UTF-16 encoding, MUST be content- transfer-encoded in base64.

In my reading it says that UTF-16 _is_ binary, which then require some additional things. So, technically, what is in the text (and which is based on the JSON registration, quite a bit simpler) is correct, and 7303 adds some other things. Maybe we could say something like

> For further details on encoding, rfc7303 also apply.

The only thing I know is that what the commenter proposed, ie, to simply use "binary", is probably wrong, because using UTF-8 is perfectly fine, too.

Reply:
I am puzzled by what the reviewer said about "Encoding considerations".  [This page](https://www.iana.org/assignments/media-types/media-types.xhtml#text) shows lots of media types derived from XML.  I think that we should follow what they do.  Here is my proposal.

Encoding considerations: Identical to those of  "application/xml" as
described in [Section 3 of [RFC7303]](https://www.rfc-editor.org/rfc/rfc7303#section-3) .

Reply:
I am sorry @murata2makoto, but we merged the PR yesterday, and I have already responded to IANA referring, and quoting from, the new text. So I'd prefer not to change the text now (we will see what they respond).

But I believe we fundamentally agree: just putting 'binary' would have been a mistake. What we have in the text now is:

> 8bit if UTF-8; binary if UTF-16.
>
> Package documents are in XML, represented either in UTF-8 or UTF-16. When the package document is written in UTF-8, the file is 8bit compatible. When it is written in UTF-16, the binary content-transfer-encoding must be used. For further details, see [[rfc7303](https://www.w3.org/TR/epub-33/#bib-rfc7303)].

Which is, I believe, essentially the same as what you propose, just a bit more verbose.